drivers/mtd/Kconfig

```diff
@@ -926,6 +926,52 @@ config RAMTRON_CHUNKING
 	---help---
 		Some Re-RAMs like MB85AS4MT have write buffer size limitation and
 		require smaller, "chunked" writes.
+		
```

Comment:
@Igor-Misic Code looks good. Can you check the line wrapping and make it consistent with the other long lines in file.